src/components/Sections/utils.ts

```diff
@@ -75,9 +75,10 @@ const sortSections = (sections: Section[]): Section[] => {
 // If no layout is provided, returns folders sorted alphabetically with the default layout
 export const formatSections = (
   folders?: DirectoryDataArray,
-  layout?: SectionSetting[] | SectionSetting
-): Section[] => {
-  if (!folders) return []
+  layout?: SectionSetting[] | SectionSetting,
+  isMobile?: boolean
+): [Section[], Section[]] => {
```

Comment:
The function documentation does not reflect this change.

Also maybe it would be safer to use an object instead of an array of array here. On the caller side it is easy to swap values.

Something like
```js
{
    groupedSections: [],
    ungroupedSections: []
}
```